test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using Xunit;
+
+namespace Microsoft.Docs
+{
+    public class YamlUtilityTest
```

Comment:
>YamlUtilityTest [](start = 17, length = 15)

add test cases covered in docfx\test\Microsoft.DocAsCode.Common.Tests\YamlSerializationTest.cs, such as
1. big integer
2. anchors support